ibtk/include/ibtk/FEDataInterpolation.h

```diff
@@ -340,9 +340,9 @@ class FEDataInterpolation
 
     // Data associated with the current element.
     const libMesh::Elem* d_current_elem = nullptr;
-    unsigned int d_current_side;
+    unsigned int d_current_side = std::numeric_limits<unsigned int>::max();
```

Comment:
Is the goal to provide initial/default values for all member variables? I gave up on doing that with C++03, but it is a lot easier now with C++11.

Reply:
Yes - it would be great to get everything initialized in the constructor to a deterministic value.